### DISCUSSION TITLE
What happened to alt-tab like feature?

### DISCUSSION THREAD

**Discussion by ex4722** [qtile/qtile#3555] [Q&A] [answered]
Hello all!

Been using qtile for almost year now and when I first started I remember this feature but it seems like it disappeared after a while. Originally opening the group that was already open would have alt-tab like feature. For example on one group hitting meta 1 would cycle to the last workspace I had opened. Is their a setting to turn this on or was this feature removed?

**Comment by elParaguayo** [comment, marked as answer]
It's still available. In your config, find the line that binds the keys to switching groups and add `toggle=True` to the command. e.g.
``` python
lazy.group[i.name].toscreen()
```
becomes
``` python
lazy.group[i.name].toscreen(toggle=True)
```

**Reply by ex4722** [reply]
Oh my god, thank you so much. I've been downgrading qtile just to get that back and each time my package manager updated it on accident I would need to downgrade again. Thank you again!!!
